Sources/Commands/SwiftTestCommand.swift

```diff
@@ -730,7 +730,7 @@ extension SwiftTestCommand {
                 try await self.runCommand(swiftCommandState)
             } catch let error as FileSystemError {
                 if sharedOptions.shouldSkipBuilding {
-                    throw ErrorWithContext(error, "Test build artifacts were not found in the build folder. Found the --skip-build flag; build the tests first or rerun the command without --skip-build")
+                    throw ErrorWithContext(error, "Test build artifacts were not found in the build folder.\n\tThe --skip-build flag was provided; either build the tests first with \"swift build --build tests\" or rerun the \"swift test list\" command without --skip-build")
```

Comment:
formatting nit: line length here clearly exceeds [120 characters specified in the formatter config](https://github.com/swiftlang/swift-package-manager/blob/d28f725139357fbb1b20148964177fef5aab3639/.swiftformat#L15C12-L15C15
), let's use a multi-line string 
```suggestion
                    throw ErrorWithContext(error, """
                        Test build artifacts were not found in the build folder.
                        The `--skip-build` flag was provided; either build the tests first with \
                        `swift build --build tests` or rerun the `swift test list` command without \
                        `--skip-build`
                        """
                    )
```

Reply:
Done